spa/src/components/content/pages/Pages.test.js

```diff
@@ -34,3 +60,52 @@ describe('Given pages list having a page with a null rp', () => {
     expect(result.length).toEqual(2);
   });
 });
+
+describe('New page button behavior given org plan and user role', () => {
+  const axiosMock = new MockAdapter(Axios);
+  afterEach(() => {
+    axiosMock.reset();
+    axiosMock.resetHistory();
+  });
+  afterAll(() => axiosMock.restore());
+  test('when user is super user can always add', async () => {
+    useUser.mockImplementation(() => ({ user: superUser }));
+    axiosMock.onGet(LIST_PAGES).reply(200, [{}, {}, {}]);
+    render(<Pages />);
+    await waitFor(() => {
+      expect(axiosMock.history.get.length).toBe(1);
+    });
+    const newPageButton = screen.getByLabelText('New Page');
+    expect(newPageButton).not.toBeDisabled();
+  });
+  test('when user is hub admin can always add', async () => {
+    useUser.mockImplementation(() => ({ user: hubAdmin }));
+    axiosMock.onGet(LIST_PAGES).reply(200, [{}, {}, {}]);
+    render(<Pages />);
+    await waitFor(() => {
+      expect(axiosMock.history.get.length).toBe(1);
+    });
+    const newPageButton = screen.getByLabelText('New Page');
+    expect(newPageButton).not.toBeDisabled();
+  });
+  test('typical self-onboarded user who has not added a page can add one', async () => {
+    useUser.mockImplementation(() => ({ user: orgAdminUser }));
+    axiosMock.onGet(LIST_PAGES).reply(200, []);
+    render(<Pages />);
+    await waitFor(() => {
+      expect(axiosMock.history.get.length).toBe(1);
+    });
+    const newPageButton = screen.getByLabelText('New Page');
+    expect(newPageButton).not.toBeDisabled();
+  });
+  test('typical self-onboarded user who has added a page cannot add one', async () => {
+    useUser.mockImplementation(() => ({ user: orgAdminUser }));
+    axiosMock.onGet(LIST_PAGES).reply(200, [{}]);
+    render(<Pages />);
+    await waitFor(() => {
+      expect(axiosMock.history.get.length).toBe(1);
+    });
+    const newPageButton = screen.getByLabelText('New Page');
```

Comment:
Would be better to use button syntax. Example:
```suggestion
    const newPageButton = screen.getByRole('button', { name: 'New Page' });
```

Reply:
76d2934845a